client/routes.jsx

```diff
@@ -9,8 +9,8 @@ import Apply from '../imports/ui/components/apply.jsx';
 import ForgotPassword from '../imports/ui/components/forgot_password.jsx';
 import SetPassword from '../imports/ui/components/set_password.jsx';
 import Dashboard from '../imports/ui/components/dashboard.jsx';
-import Home from '../imports/ui/components/home.jsx';
-import Moderator from '../imports/ui/components/home.jsx';
+// import Home from '../imports/ui/components/home.jsx';
```

Comment:
I don't believe home actually exists? Can we just delete it instead of commenting it out?

Reply:
Yes, thanks for catching!